### PR TITLE
Novacustom test matrix update

### PR DESCRIPTION
**Pull request by johanes2115**

### CODE REVIEW

**Comment by miczyg1** [comment]
@johanes2115 this PR needs rebase, changes to the hooks are already on master

File: docs/variants/novacustom_ns5x_tgl/test-matrix.md [hunk 1]
```diff
@@ -37,9 +37,13 @@ subjected before the release of the new binary.
...
 | 27.  | [QubesOS support][QBS]                                 | QBS           | All                                  |
 | 28.  | [Fedora support][FED]                                  | FED           | All                                  |
-| 29.  | [Platform suspend and resume][SUSP]                    | SUSP          | SUSP001.001, SUSP002.001, SUSP003.001 |
-| 30.  | [Sign of life][SOL]                                    | SOL           | All                                  |
-| 31.  | [BIOS menu function keys][BMF]                         | BMF           | All                                  |
+| 29.  | [Platform suspend and resume][SUSP]                    | SUSP          | Without SUSP004.001 and SUSP006.001  |
+| 30.  | [Boot blocking][BBB]                                   | BBB           | All                                  |
+| 31.  | [Reset to defaults][RTD]                               | RTD           | All                                  |
+| 32.  | [Suspend mechanism switchcing (S0ix/S3)][SMS]           | SMS           | All                                 |
```
Comment (miczyg1) [inline docs/variants/novacustom_ns5x_tgl/test-matrix.md:43]:
```suggestion
| 32.  | [Suspend mechanism switchcing (S0ix/S3)][SMS]          | SMS           | All                                 |
```

File: docs/variants/novacustom_nv4x_tgl/test-matrix.md [hunk 2]
```diff
@@ -117,88 +136,10 @@ subjected from before the release of the new binary.
...
 | 2.   | [M.2 Wi-fi][SMW]                                  | SMW           | All                                  |
 | 3.   | [NVMe detection][SNV]                             | SNV           | All                                  |
+| 4.   | [NET interface detection][NET]                    | NET           | All                                  |
+| 5.   | [TPM detection][TPD]                              | TPD           | TPD003.001, TPD004.001               |
 
 [SUD]: ../../unified-test-documentation/dasharo-stability/C01-usb-type-a-devices-detection.md
 [SMW]: ../../unified-test-documentation/dasharo-stability/C02-m2-wi-fi.md
 [SNV]: ../../unified-test-documentation/dasharo-stability/C03-nvme-detection.md
-
-## NV41MZ test matrix
```
Comment (miczyg1) [inline docs/variants/novacustom_nv4x_tgl/test-matrix.md:125]: Why do we completely remove NV41MZ?

File: docs/variants/novacustom_ns5x_adl/test-matrix.md [hunk 3]
```diff
@@ -38,7 +38,12 @@ subjected before the release of the new binary.
 | 27.  | [QubesOS support][QBS]                                 | QBS           | All                                  |
 | 28.  | [Fedora support][FED]                                  | FED           | All                                  |
 | 29.  | [Platform suspend and resume][SUSP]                    | SUSP          | Without SUSP004.001 and SUSP006.001  |
-| 30.  | [BIOS menu function keys][BMF]                         | BMF           | All                                  |
+| 30.  | [Boot blocking][BBB]                                   | BBB           | All                                  |
+| 31.  | [Reset to defaults][RTD]                               | RTD           | All                                  |
+| 32.  | [Suspend mechanism switchcing (S0ix/S3)][SMS]           | SMS           | All                                 |
```
Comment (miczyg1) [inline docs/variants/novacustom_ns5x_adl/test-matrix.md:43]:
```suggestion
| 32.  | [Suspend mechanism switchcing (S0ix/S3)][SMS]          | SMS           | All                                 |
```

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: docs/variants/novacustom_ns5x_tgl/test-matrix.md (hunk 1 above)
Comment (johanes2115) [reply docs/variants/novacustom_ns5x_tgl/test-matrix.md]: Fixed in 4856d86caf7b885bf1645c10375a7d999fd4ab71

File: docs/variants/novacustom_ns5x_adl/test-matrix.md (hunk 3 above)
Comment (johanes2115) [reply docs/variants/novacustom_ns5x_adl/test-matrix.md]: Fixed in 4856d86caf7b885bf1645c10375a7d999fd4ab71

Comment (johanes2115) [reply docs/variants/novacustom_ns5x_adl/test-matrix.md]: Had to do that by mistake, fixed in 53b374c64145ec09e899ea4854ae1874ca8a0473

File: docs/variants/novacustom_nv4x_tgl/test-matrix.md (hunk 2 above)
Comment (johanes2115) [reply docs/variants/novacustom_nv4x_tgl/test-matrix.md:125]: Had to do that by mistake, fixed in 53b374c64145ec09e899ea4854ae1874ca8a0473

**Review by miczyg1 — APPROVED** [review]

**Comment by pietrushnic** [comment]
@miczyg1 by merging this, we lost 51/100 (change of rank from 99 to 48) points in the SEO rank of docs.dasharo.com. The key problem here was accepting file name changes without reflecting that in mkdocs.yml

@Asiderr why CI didn't block that !?

**Pull request merged**